test/e2e/labels_test.go

```diff
@@ -4,6 +4,7 @@
 package e2e
 
 import (
+	"github.com/aws/eks-anywhere/pkg/constants"
```

Comment:
I think this import should be lower, with the other github imports

Reply:
code refactored to other places.